src/containers/AppLoader/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { View, ActivityIndicator, StyleSheet } from 'react-native';
+import { object } from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import useSessionChangeEffect from 'hooks/useSessionChangeEffect';
+import { LOGIN_SCREEN, MAIN_SCREEN } from 'constants/screens';
+
+const styles = StyleSheet.create({
```

Comment:
Should this style be here?

Reply:
I assumed he did this because this is not the purpose of the draft, but if we're gonna add this to the base it should be in a styles file of course